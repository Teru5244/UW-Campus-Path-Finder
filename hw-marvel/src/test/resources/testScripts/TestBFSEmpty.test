# Test BFS on an empty graph

LoadGraph empty empty.csv

FindPath empty n1 n2

FindPath empty you me

FindPath empty Failure Success