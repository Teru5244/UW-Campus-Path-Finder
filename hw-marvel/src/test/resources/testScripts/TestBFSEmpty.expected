# Test BFS on an empty graph

loaded graph empty

unknown: n1
unknown: n2

unknown: you
unknown: me

unknown: Failure
unknown: Success